packages/react-components/src/components/TypingIndicator.test.tsx

```diff
@@ -9,14 +9,74 @@ import { mountWithLocalization, createTestLocale } from './utils/testUtils';
 
 Enzyme.configure({ adapter: new Adapter() });
 
+const randomText = Math.random().toString();

```

Comment:
Is there a reason to use randomText and not a fixed test string? The issue with randomText is if we ever did a `expect().toMatchSnapshot` it would break. (and in general it's best to avoid math.random in tests as it introduces a tiny amount of uncertainty which could lead to flakey tests further down the line)

Reply:
(Oh I see this line already existsed and was just moved - feel free to not resolve this comment)

Reply:
not particularly. I just extended the existing test.
I will change that :)